docker/base

```diff
@@ -55,5 +55,32 @@ gem install $(gem_ver travis "$VERSION_TRAVIS")
 gem install $(gem_ver fpm "$VERSION_FPM")
 
 # Get jfrog tool from Bintray
-curl -XGET -L -k "https://api.bintray.com/content/jfrog/jfrog-cli-go/${VERSION_JFROG}/jfrog-cli-linux-amd64/jfrog?bt_package=jfrog-cli-linux-amd64" > /usr/local/bin/jfrog
-chmod a+x /usr/local/bin/jfrog
+download_jfrog()
+{
+	{ set +x } > /dev/null
+	local ver="$1"
+	local dst="/usr/local/bin/jfrog"
+
+	# Get sha256 sum of the file
+	local json=$(set -x; curl -sLkX GET \
+		"https://api.bintray.com/packages/jfrog/jfrog-cli-go/jfrog-cli-linux-amd64/versions/$ver/files")
+	# From the received array, select the object with the specific "path"
+	# attribute, and from that object get the "sha256" attribute
+	local sha256=$(echo "$json" | jq -r ".[] |
+			select(.path == \"$ver/jfrog-cli-linux-amd64/jfrog\") |
+			.sha256")
+
+	# Get the file
+	(set -x; curl -#LkX GET \
+		"https://dl.bintray.com/jfrog/jfrog-cli-go/$ver/jfrog-cli-linux-amd64/jfrog" \
+		> "$dst")
+
+	# Check file's sha256 sum
+	local sha256new="$(sha256sum "$dst" | cut -c1-64)"
```

Comment:
True (it's actually more like `echo "$sha256  $dst" | sha256sum -c -`) but this gives a better debug output. The check will say just if it failed or succeeded but if it fails it won't show the gotten and expected sums. So I prefer slightly the more verbose version.